ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -25,6 +30,12 @@ export const CronWorkflowDetails = ({match, location, history}: RouteComponentPr
     const [name] = useState(match.params.name);
     const [sidePanel, setSidePanel] = useState(queryParams.get('sidePanel'));
     const [tab, setTab] = useState(queryParams.get('tab'));
+    const [workflows, setWorkflows] = useState<Workflow[]>([]);
+    const [{selectedPhases, selectedLabels, paginationLimit}, setSelectedData] = useState<WorkflowListRenderOptions>(JSON.parse(localStorage.getItem('ListOptions/options')));
```

Comment:
there are no phases or labels here. those are from the sidebar filter and the Workflows list which is not present here.

pagination does not seem to be present either. there seems to be some logic here, but no UI elements for pagination.

also the `localStorage` call would not apply here as it is specific to the Workflows list page

Reply:
- Thanks I was wondering this, would it be a good idea to put a selection or pagination in the detail cron page?

Reply:
I don't think filtering is needed right now. pagination would be nice to have, but for a first-pass MVP, I think we can leave out pagination as well.

You may have not noticed, but the Cron Workflows page itself actually doesn't implement pagination yet 😅 : #10846